### ISSUE TITLE
cuckoo api - 500 Internal Server Error

### ISSUE THREAD

**Issue by TiTi87** [cuckoosandbox/cuckoo#1665]
Hello everybody,

I'm getting an "error 500" on Cuckoo API while aksing for `/task/list` on my cuckoo-dev up to date with the lastet github source.

> Internal Server Error
> 
> The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.

Cuckoo error logs seems pretty direct about some encoding/decoding error :

```
--> 2017-06-16 09:01:54,621 [cuckoo.apps.api] ERROR: Exception on /tasks/list [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/Cuckoo-2.0.3-py2.7.egg/cuckoo/apps/api.py", line 237, in tasks_list
    task["target"] = task["target"].decode("latin-1")
UnicodeEncodeError: 'ascii' codec can't encode character u'\xb0' in position 27: ordinal not in range(128)
2017-06-16 09:01:54,621 [werkzeug] INFO: 192.168.2.1 - - [16/Jun/2017 09:01:54] "GET /tasks/list HTTP/1.1" 500 -
```
Thanks in advance !

**Comment by jbremer** [comment]
Hi, thanks for the feedback. This is a duplicate of #1553. We'll try to address the issue soon'ish! :)

**Comment by jbremer** [comment]
This issue has now been fixed in the commit above. This will be taken into the next alpha or stable release! :)

**Comment by jbremer** [comment]
We've just released `2.0.4a3` which includes the fix mentioned above.
